src/app/app.module.ts

```diff
@@ -76,6 +76,7 @@ import { DirectivesModule } from './directives/directives.module';
 import { ToggleInfoDrawerComponent } from './components/toolbar/toggle-info-drawer/toggle-info-drawer.component';
 import { DocumentDisplayModeComponent } from './components/toolbar/document-display-mode/document-display-mode.component';
 import { ToggleFavoriteComponent } from './components/toolbar/toggle-favorite/toggle-favorite.component';
+import { ContextmenuModule } from './contextmenu/contextmenu.module';
```

Comment:
Can we have it named `ContextMenuModule` and `context-menu` for file names to align with angular style guides?